nrf_802154_sl/CHANGELOG.rst

```diff
@@ -3,12 +3,19 @@
 Changelog
 #########
 
-.. contents::
```

Comment:
Should this part be removed? I thought it was added to unify with other libraries.